db/migrate/20170619161514_add_options_to_ext_management_system.rb

```diff
@@ -0,0 +1,34 @@
+class AddOptionsToExtManagementSystem < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options
+  end
+  class CustomAttribute < ActiveRecord::Base
+    serialize :serialized_value
+  end
+
+  def up
+    add_column :ext_management_systems, :options, :text
+
+    say_with_time("Migrating Kubernetes provider options") do
+      ExtManagementSystem.where(:type => ["ManageIQ::Providers::Kubernetes::ContainerManager",
```

Comment:
I think we can remove N+1 and do something like:
```ruby
ExtManagementSystem.where(...).includes(:custom_attributes).where(:custom_attributes => {:section=> 'cluster_settings',... }.each do |ems|
options = ems.custom_attributes.map { |ca| Hash[*["image_inspector_#{ca.name}", ca.value]] }
ems.update(:options => options)
end
``` 
Does it cover your intention?